flow/BUILD.gn

```diff
@@ -12,6 +12,12 @@ source_set("flow") {
     "compositor_context.h",
     "diff_context.cc",
     "diff_context.h",
+    "display_list.cc",
```

Comment:
Minor nit: Can we put the ops in their own header? The stuff about the display list is way more interesting that the op definitions. Will just make the code a bit easier to navigate.

Reply:
Currently they are private to that cc file. Their definitions aren't needed anywhere else.

Basically `display_list.[h,cc]` exist only to define, pack, and unpack the structure through Dispatcher. Everything else is in a different file.

Reply:
I'll lump this in with the previous suggestions for using a manifest and for investigating alternate storage solutions.